kroxylicious-kubernetes-api/src/main/resources/META-INF/fabric8/kafkaprotocolfilters.filter.kroxylicious.io-v1.yml

```diff
@@ -69,7 +69,7 @@ spec:
                     The metadata.generation that was observed during the last reconciliation by the operator.
                   type: integer
                 conditions:
-                  # Mapped to Java type io.kroxylicious.kubernetes.api.common.Condition
+                  # Mapped to Java type io.kroxylicious.kubernetes.api. io.kroxylicious.kubernetes.api.common.common.Condition
```

Comment:
I suspect that something's gone wrong with a search/replacement resulting in this comment being screwed.

Reply:
Oh good spotting